lib/administrate/field/belongs_to.rb

```diff
@@ -8,7 +8,7 @@ def self.permitted_attribute(attr)
       end
 
       def permitted_attribute
-        self.class.permitted_attribute(attribute)
+        foreign_key || self.class.permitted_attribute(attribute)
```

Comment:
So, I found this whole `permitted_attribute` instance method calling a class method thing to be hard to follow and reason about. How would you feel about renaming class level implementations of `permitted_attribute` to like `default_attribute` or something like that? Or we could add an instance level `default_attribute` method and put that here instead of calling a class method as a default?